reference/fleet/secure-logstash-connections.md

```diff
@@ -1,6 +1,8 @@
 ---
 mapped_pages:
   - https://www.elastic.co/guide/en/fleet/current/secure-logstash-connections.html
+applies_to:
+  stack: ga
```

Comment:
I think [this page](https://docs-v3-preview.elastic.dev/elastic/docs-content/pull/3238/reference/fleet/secure-logstash-connections) is applicable to serverless and other deployment methods.

certutil is called out as being unavailable on cloud, but the logstash output is not unique to self-managed AFAIK